app/modules/names/models.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Names database models
```

Comment:
Might be helpful to explain what this is meant to be a name for. 
That it's not some generic thing for naming, e.g could be applied as name of an organisation etc